src/main/java/io/seqera/tower/cli/responses/data/DataLinkMessageResponse.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2021-2023, Seqera.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package io.seqera.tower.cli.responses.data;
+
+import io.seqera.tower.cli.responses.Response;
+
+public class DataLinkMessageResponse extends Response {
```

Comment:
minor: Might be worth renaming this class to state that it is used as a response for the delete command :)